backend/tarsy/integrations/llm/client.py

```
@@ -25,22 +33,63 @@
+
+def _create_google_client(temp, api_key, model, disable_ssl_verification=False, base_url=None):
+    """Create ChatGoogleGenerativeAI client."""
+    client_kwargs = {
+        "model": model, 
+        "temperature": temp, 
+        "google_api_key": api_key
+    }
+    # Note: ChatGoogleGenerativeAI may not support custom base_url or HTTP clients
+    return ChatGoogleGenerativeAI(**client_kwargs)
+
+def _create_xai_client(temp, api_key, model, disable_ssl_verification=False, base_url=None):
```

Ruff (0.12.2)

Unused function argument: `disable_ssl_verification`

(ARG001)